src/main/java/android/util/Base64.java

```diff
@@ -0,0 +1,741 @@
+/*
+ * Copyright (C) 2010 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package android.util;
```

Comment:
Yup, I know that it's port from Android and it's not the framework dependency. I'm only talking about package naming convention. It's strange when in pure java project we have package 'android'.  And I'm sure that you can rename package name ;)
And obviously I know what the reason is because I have the same problem on Android. I suggested to copy into this library implementation of Base64 from java8. And it will be working too because its only encrypting algorithm. Without any specyfice java8 mechanism.

It's only my recommendation to just copy Java8 code into project and add it to 'svarzee.gps.gpsoauth.util' pacakge. And of course all licence information should be included in Base64 java file too.